internal/oidc/staticclient/pinnipedcli.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package staticclient
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/ory/fosite"
+	"gopkg.in/square/go-jose.v2"
+)
+
+const ClientIDPinnipedCLI = "pinniped-cli"
+
+type PinnipedCLI struct{}
+
+// Implement methods of fosite.Client:
+var _ fosite.Client = &PinnipedCLI{}
+
+func (c *PinnipedCLI) GetID() string             { return ClientIDPinnipedCLI }
```

Comment:
```suggestion
func (c PinnipedCLI) GetID() string             { return ClientIDPinnipedCLI }
```

Not sure pointer receiver on empty struct makes sense.